src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -171,7 +172,28 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 			}
 
 			if (OperatingSystem.IsAndroidVersionAtLeast(28))
+			{
+				var fontWeightMap = new Dictionary<string, FontWeight>
+				{
+					{ "-light", FontWeight.Light },
+					{ "-medium", FontWeight.Medium },
+					{ "-black", FontWeight.Black }
+					// Add more styles as needed
+				};
+
+				if (!string.IsNullOrWhiteSpace(fontFamily))
+				{
+					foreach (var fontWeight in fontWeightMap.Keys)
```

Comment:
Can you iterate over `fontWeightMap` instead? Instead of `.Keys`?

Reply:
Sure!